packages/types/src/sights.ts

```diff
@@ -114,6 +114,21 @@ export enum VehicleModel {
   VWTROC = 'vwtroc',
 }
 
+/**
+ * A mapping of vehicle types to vehicle models.
+ */
+export const VehicleTypeMapVehicleModel: Record<VehicleType, VehicleModel> = {
+  [VehicleType.HATCHBACK]: VehicleModel.AUDIA7,
+  [VehicleType.CUV]: VehicleModel.FESC20,
+  [VehicleType.PICKUP]: VehicleModel.FF150,
+  [VehicleType.CITY]: VehicleModel.FFOCUS18,
+  [VehicleType.VAN]: VehicleModel.FTRANSIT18,
+  [VehicleType.SEDAN]: VehicleModel.HACCORD,
+  [VehicleType.LARGE_SUV]: VehicleModel.JGC21,
+  [VehicleType.MINIVAN]: VehicleModel.TSIENNA20,
+  [VehicleType.SUV]: VehicleModel.VWTROC,
+};
```

Comment:
You don't need this map. In the `vehicles` object exported by the sights package, each vehicle model has a `type` property that corresponds to the vehicle type. In order to find the vehicle model using the vehicle type, you simply find the vehicle model that has the given vehicle type. The only little thing is that there are two vehicles with `hatchback` type : forget about `audia7` (you can filter `audia7` from the list of vehicle models before searching) and only keep `ffocus18`